shared/actions/router.js

```diff
@@ -1,42 +1,5 @@
 import * as Constants from '../constants/router'
 
-/*
```

Comment:
i think the problem is the actions and reducers are a global thing and so we don't have anything that guarantees that only a thing inside of that one tab will be doing things within itself. we also have some patterns where we'll asyc do things with the server and want to make changes (like you remove a device from the devices page, if you're looking at that device, it should go up a level and refresh. it should do this even if you switched tabs). I think we can leave the tab optional since its a rarer case. basically any async action which does a navigation i think has to have the tab and extra logic to ensure the view hierarchy isn't messed up (being the same tab isn't enough of a guarantee)